src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -570,6 +570,7 @@ def addError(self, stepName, exitCode, errorType, errorDetails):
 
         stepSection = self.retrieveStep(stepName)
 
+        errorCount = getattr(stepSection.errors, "errorCount", 0)
```

Comment:
Were you trying to add some other line here? It's the same as the one below.

Reply:
My mistake, I will remove that.